From 5075bbd8e2588aea9b7f7cc3b7562f1a73d67137 Mon Sep 17 00:00:00 2001
From: Jiang Lu <lu.jiang@windriver.com>
Date: Thu, 4 Jul 2013 13:56:05 +0800
Subject: [PATCH 273/430] ppc:Cornet_DS:remove fman_bman early init routine

Remove the code introduced by patch "fman: introduce fman_init_early
function"

Signed-off-by: Jiang Lu <lu.jiang@windriver.com>
---
 arch/powerpc/platforms/85xx/corenet_ds.c |    6 ------
 1 files changed, 0 insertions(+), 6 deletions(-)

diff --git a/arch/powerpc/platforms/85xx/corenet_ds.c b/arch/powerpc/platforms/85xx/corenet_ds.c
index 75c0569..527b084 100644
--- a/arch/powerpc/platforms/85xx/corenet_ds.c
+++ b/arch/powerpc/platforms/85xx/corenet_ds.c
@@ -154,9 +154,6 @@ int __init declare_of_platform_devices(void)
 /* Early setup is required for large chunks of contiguous (and coarsely-aligned)
  * memory. The following shoe-horns Qman/Bman "init_early" calls into the
  * platform setup to let them parse their CCSR nodes early on. */
-#ifdef CONFIG_FSL_FMAN
-void __init fman_init_early(void);
-#endif
 #ifdef CONFIG_FSL_QMAN_CONFIG
 void __init qman_init_early(void);
 #endif
@@ -169,9 +166,6 @@ void __init pme2_init_early(void);
 
 __init void corenet_ds_init_early(void)
 {
-#ifdef CONFIG_FSL_FMAN
-	fman_init_early();
-#endif
 #ifdef CONFIG_FSL_QMAN_CONFIG
 	qman_init_early();
 #endif
-- 
1.7.5.4

